playbooks/common/openshift-cluster/upgrades/v3_0_to_v3_1/upgrade.yml

```diff
@@ -235,6 +238,11 @@
 
   - name: Upgrade master packages
     command: "{{ ansible_pkg_mgr}} update -y {{ openshift.common.service_type }}-master{{ openshift_version }}"
+    when: openshift_pkg_version is not defined
+
+  - name: Upgrade packages
+    command: "{{ ansible_pkg_mgr}} install -y {{ openshift.common.installed_product_rpms | oo_31_rpm_rename_conversion(openshift_version) | join (' ')}}"
```

Comment:
@detiber, For some reason installed_product_rpms isn't set here.  If I put a debug line at near the top of this playbook it's set.  Any idea what's going on?